src/robinhood.py

```diff
@@ -71,25 +69,43 @@ def get_ticker(cls, ticker):
         Robinhood.check_login_time()
 
         fundamentals = r.get_fundamentals(ticker)
+        instrument_data = r.find_instrument_data(ticker)
+
+        # Detect invalid index
         if fundamentals[0] == None:
             return False
 
         ticker_data = {}
+        ticker_data["market"] = instrument_data[0]["market"].strip("https://api.robinhood.com/markets/")
         ticker_data["current"] = round(float(r.stocks.get_latest_price(ticker)[0]),2)
         ticker_data["volume"] = round(float(fundamentals[0]["volume"]),2)
         ticker_data["avg_volume"] = round(float(fundamentals[0]["average_volume"]), 2)
         ticker_data["market_cap"] = round(float(fundamentals[0]["market_cap"]), 2)
-        ticker_data["market_status"] = 0
-        ticker_data["name"] = r.stocks.find_instrument_data(ticker)[0]['simple_name']
-        ticker_data['points_change'] = 0
+        ticker_data["name"] = r.stocks.find_instrument_data(ticker)[0]["simple_name"]
```

Comment:
If we are not going to use this variable on the front-end, there's no need to add it to the JSON payload. You can just save it to a variable.

Reply:
Isn't the name used on the frontend to display the corp name below the stock index?

Reply:
I accidentally highlighted the wrong line, I am referring to the `market` key.

Reply:
Removed from json in next commit